<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>483</height>
     </rect>
    </property>
    <widget class="QLabel" name="bg">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>960</width>
       <height>483</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../resources/bg.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>960</width>
       <height>483</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>961</width>
        <height>101</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="title">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>-10</y>
         <width>461</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../resources/taskbuddy.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>221</width>
        <height>131</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="Create_task">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>170</width>
         <height>90</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Variable Display Semib</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: transparent;
    border: 2px solid black;
    border-radius: 25px;
}

QPushButton:hover {
    background-color: rgba(0, 0, 255, 50);
    border: 2px solid #000080;
    transition: background-color 0.3s, border 0.3s; 
}</string>
       </property>
       <property name="text">
        <string>Enter</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../resources/plus symbol.png</normaloff>../resources/plus symbol.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>110</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>100</y>
        <width>241</width>
        <height>141</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="Settings">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>201</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Variable Display Semib</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: transparent;
    border: 2px solid black;
    border-radius: 25px;
}

QPushButton:hover {
    background-color: rgba(0, 0, 255, 50);
    border: 2px solid #000080;
    transition: background-color 0.3s, border 0.3s; 
}</string>
       </property>
       <property name="text">
        <string>Settings</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../resources/gear.png</normaloff>../resources/gear.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>130</y>
        <width>451</width>
        <height>351</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QListWidget" name="task_list">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>401</width>
         <height>271</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
</string>
       </property>
      </widget>
      <widget class="QLabel" name="task_list_title">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>381</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Variable Small Semibol</family>
         <pointsize>22</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Today's Mission :</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_7">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>260</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="Completed">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Variable Display Semib</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: transparent;
    border: 2px solid black;
    border-radius: 25px;
}

QPushButton:hover {
    background-color: rgba(0, 0, 255, 50);
    border: 2px solid #000080;
    transition: background-color 0.3s, border 0.3s; 
}</string>
       </property>
       <property name="text">
        <string>Completed</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../resources/todolisticon.png</normaloff>../resources/todolisticon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>141</width>
         <height>51</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>201</width>
        <height>211</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="entertask">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>171</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Variable Display Semib</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Enter the task :</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="task_input">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Variable Display</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI Variable Display'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:7.8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
